fia_api/core/auth/tokens.py

```diff
@@ -80,3 +85,15 @@ def _is_jwt_access_token_valid(self, access_token: str) -> bool:
         except RuntimeError:  # pylint: disable=broad-exception-caught)
             logger.exception("Error decoding JWT access token")
             return False
+
+    def _is_api_key_valid(self, api_key: str) -> bool:
+        """
+        Check if the JWT access token is valid.
+
+        It does this by checking that it was signed by the corresponding private key and has not expired. It also
+        requires the payload to contain a username.
+        :param api_key: The JWT access token to check.
+        :return: `True` if the JWT access token is valid and its payload contains a username, `False` otherwise.
+        """
```

Comment:
Docstring is talking about JWT instead of API key.

Reply:
Good spot!